packages/pipeline-editor/src/PipelineExportDialog.tsx

```diff
@@ -48,7 +49,7 @@ export class PipelineExportDialog extends React.Component<IProps, IState> {
   state = {
     displayedRuntimeOptions: new Array<IRuntime>(),
     fileTypes: new Array<Record<string, string>>(),
-    selectedRuntimePlatform: '',
+    selectedRuntimePlatform: this.props.runtime?.name || '',
```

Comment:
I'm a little confused where this state is being use? Seems like it's only ever accessed in componentDidMount?

Also, I would use nullish coalescing if possible: `this.props.runtime?.name ?? ''`

Reply:
Oh good catch, not sure why exactly but I'm suspecting it's from copying over code from `PipelineSubmissionDialog`. Just removed it and it's working as expected.